### ISSUE TITLE
[Simple AA in NewDot] No option to leave workspace as  non-owner

### ISSUE THREAD

**Issue by m-natarajan** [Expensify/App#43508]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.81-8
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @Beamanator 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1718092341741619

## Action Performed:
1. As `User A`, create policy in NewDot 
2. Invite two other users. Promote `User B` to workspace admin, and leave `User C` as a regular user
3. Log in as `User B`
4. Open workspace members page & try to leave the workspace
5. Log in as `User C`
6. Open workspace members page & try to leave the workspace
Expected Result: Both `User B` and `User C` should be able to leave the workspace

## Expected Result:
There should be a way for a workspace non-owner to leave a workspace.

## Actual Result:
There's no way for either of the workspace non-owners to leave the workspace.

## Workaround:
unknown
## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [x] Android: mWeb Chrome
- [x] iOS: Native
- [x] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [x] MacOS: Desktop

## Screenshots/Videos

https://github.com/Expensify/App/assets/38435837/6a16587a-77da-48ab-a20a-505461c63013

![Snip - (60) New Expensify - Google Chrome](https://github.com/Expensify/App/assets/38435837/04474eb2-f688-4d8d-a52a-7223cfc85d1c)

</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by truph01** [comment]
## Proposal

### Please re-state the problem that we are trying to solve in this issue.
- There should be a way for a non-owner workspace admin to leave a workspace.

### What is the root cause of that problem?
- If the current user is admin, we always display button:
https://github.com/Expensify/App/blob/8ed1e042af77ac0674941db9c80b67979426f7e8/src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx#L186-L194

- And the above button is disabled because of the condition:
https://github.com/Expensify/App/blob/8ed1e042af77ac0674941db9c80b67979426f7e8/src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx#L190
### What changes do you think we should make in order to solve the problem?
- We can display the Button having the title "Leave" and when pressing on it, call ```Policy.leaveWorkspace(policyID ?? '')```. This button is only displayed if current user is admin. To do that, we can update logic:
- https://github.com/Expensify/App/blob/8ed1e042af77ac0674941db9c80b67979426f7e8/src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx#L185-L195
to:
```
) : isCurrentUserAdmin && !isCurrentUserOwner ? (
                            <Button
                                text={'Leave'}
                                onPress={() => Policy.leaveWorkspace(policyID)}
                                medium
                                icon={Expensicons.RemoveMembers}
                                iconStyles={StyleUtils.getTransformScaleStyle(0.8)}
                                style={styles.mv5}
                            />
                        ) : (
                            <Button
                                text={translate('workspace.people.removeMemberButtonTitle')}
                                onPress={askForConfirmationToRemove}
                                medium
                                isDisabled={isSelectedMemberOwner || isSelectedMemberCurrentUser}
                                icon={Expensicons.RemoveMembers}
                                iconStyles={StyleUtils.getTransformScaleStyle(0.8)}
                                style={styles.mv5}
                            />
                        )}
```

### What alternative solutions did you explore? (Optional)

**Comment by truph01** [comment]
## Proposal
  
### Please re-state the problem that we are trying to solve in this issue.
- There should be a way for a non-owner workspace admin to leave a workspace.

### What is the root cause of that problem?
- If the current user is admin, we always display button:
https://github.com/Expensify/App/blob/8ed1e042af77ac0674941db9c80b67979426f7e8/src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx#L186-L194

- And the above button is disabled because of the condition:
https://github.com/Expensify/App/blob/8ed1e042af77ac0674941db9c80b67979426f7e8/src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx#L190
### What changes do you think we should make in order to solve the problem?
- We can display the Button having the title "Leave" and when pressing on it, call ```Policy.leaveWorkspace(policyID ?? '')```. This button is only displayed if current user is admin and not a owner. To do that, we can update logic:
- https://github.com/Expensify/App/blob/8ed1e042af77ac0674941db9c80b67979426f7e8/src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx#L185-L195
to:
```
) : isCurrentUserAdmin && !isCurrentUserOwner ? (
                            <Button
                                text={'Leave'}
                                onPress={() => Policy.leaveWorkspace(policyID)}
                                medium
                                icon={Expensicons.RemoveMembers}
                                iconStyles={StyleUtils.getTransformScaleStyle(0.8)}
                                style={styles.mv5}
                            />
                        ) : (
                            <Button
                                text={translate('workspace.people.removeMemberButtonTitle')}
                                onPress={askForConfirmationToRemove}
                                medium
                                isDisabled={isSelectedMemberOwner || isSelectedMemberCurrentUser}
                                icon={Expensicons.RemoveMembers}
                                iconStyles={StyleUtils.getTransformScaleStyle(0.8)}
                                style={styles.mv5}
                            />
                        )}
```

- Then BE need to fix the ```LeavePolicy``` API so that non owner admin can leave workspace.
### What alternative solutions did you explore? (Optional)

- With the member of the workspace who is not admin or owner, we also need to display "Leave" option which allows them to leave workspace as well because we already displayed "Leave" option in WorkspaceListPage if user is not admin and owner:
https://github.com/Expensify/App/blob/8d11d0b185083a6a813daeb0fe7c0697a5627ef6/src/pages/workspace/WorkspacesListPage.tsx#L173-L179

**Comment by Beamanator** [comment]
FYI we haven't added the `Help Wanted` label yet b/c we're still designing out exactly how we want this to work, so I wouldn't recommend adding proposals yet 😅

**Comment by johncschuster** [comment]
Thanks for the update, @Beamanator. In that case, should this be `Daily`?

**Comment by johncschuster** [comment]
@Beamanator, did the discussion land on this?

**Comment by Beamanator** [comment]
Fair, maybe weekly is better 😅  I have been OOO yesterday and most of last friday so i have some stuff to catch up on, but the latest should be in the linked slack thread! I can try to look at this today soonish as well :D

**Comment by johncschuster** [comment]
Discussion is still ongoing in Slack (looks like we're close to a path forward).

Setting this to `Weekly` for now.

**Comment by johncschuster** [comment]
I _think_ this fits under #wave-collect. @trjExpensify, would you agree?

**Comment by trjExpensify** [comment]
Did we conclude on this? https://github.com/Expensify/App/issues/43508#issuecomment-2163019303

Seems like it would be something we put in #wave-control and track with that to solve allowing anyone that's not the owner to freely leave a workspace.

**Comment by johncschuster** [comment]
> Did we conclude on this? https://github.com/Expensify/App/issues/43508#issuecomment-2163019303

I think we did! @Beamanator, as far as I can tell, [this comment](https://expensify.slack.com/archives/C049HHMV9SM/p1718868345497049?thread_ts=1718092341.741619&cid=C049HHMV9SM) is the path forward, right?

**Comment by Beamanator** [comment]
Nice, ya i think so!

So for the `External` part of this issue, it looks like basically just want to:
1. Give non-owner admins the ability to Leave workspaces
2. The "Leave" button should be in the 3-dot menu

Now the possibly interesting thing is: Once we implement this being possible, if a non-owner admin in a Control workspace see a warning about themselves being replaced with the Owner, in every member's policy chain? I kinda assume not, we'll just do that automatically?

**Comment by johncschuster** [comment]
@trjExpensify ^^

Given we've come to a conclusion, how about we switch this back to `Daily` and throw on `Help Wanted`?

**Comment by trjExpensify** [comment]
Woof, spotted another thing. I'm an admin non-owner on this workspace and I see the `Delete workspace` option in NewDot. Only the owner should be able to delete the workspace... 😬 

<img width="1138" alt="image" src="https://github.com/user-attachments/assets/04587efb-68cb-4479-b940-5cccc1366fc7">

<img width="1091" alt="image" src="https://github.com/user-attachments/assets/5eedf658-86b0-437e-a2a0-200b9aaafb6b">

**Comment by trjExpensify** [comment]
>Now the possibly interesting thing is: Once we implement this being possible, if a non-owner admin in a Control workspace see a warning about themselves being replaced with the Owner, in every member's policy chain? I kinda assume not, we'll just do that automatically?

Are we implementing that as part of the approvals logic you're working on? That's why I was kinda' asking whether this should go in #wave-control and be tracked with that project, so it's implemented in tandem.

**Comment by Beamanator** [comment]
> Are we implementing that as part of the approvals logic you're working on? That's why I was kinda' asking whether this should go in #wave-control and be tracked with that project, so it's implemented in tandem.

Not separately, this should probably all be handled in this issue

**Comment by trjExpensify** [comment]
Sorry man, what's the tangible difference between a non-owner admin and a member leaving a workspace in this regard? Both user types can be a configured as an approver on the workspace (i.e a configured submitsto, forwardsTo, overLimitForwardsTo). So if we've done the work for the latter to let members leave a workspace as [mentioned here](https://github.com/Expensify/App/issues/42555#issuecomment-2186106530), what do we need to do differently in this issue for the warning?

**Comment by Beamanator** [comment]
I was basically just asking if we need any difference in warning messages for admins vs non-admins leaving the workspace :D If not, groovy!

**Comment by johncschuster** [comment]
Not overdue. `Hot Pick` was just recently added to the issue. Let's give it a couple days to see if it gets picked up.

**Comment by trjExpensify** [comment]
>I was basically just asking if we need any difference in warning messages for admins vs non-admins leaving the workspace

Can you share the warning message for an admin?

**Comment by Beamanator** [comment]
OK so if I understand correctly, here’s what we want:
1. Leaving a workspace
    1. Collect
        1. NOT ALLOWED if You’re the current manager of any report (a.k.a. if a report has been submitted to you and you’re in line to approve it)
            1. However, admin can remove you in this case. If you’re the default approver, admin will see warning that you’ll be replaced by owner?
        2. Otherwise, if you’re the default approver, show warning “Owner will replace you as default approver”
    2. Control
        1. NOT ALLOWED if You’re the current manager of any report
            1. However, admin can remove you in this case. If you’re the `submitsTo` or `forwardsTo` for anyone in the policy, show warning to admin that you’ll be replaced w/ policy owner
        2. Otherwise, if you’re the `submitsTo` or `forwardsTo` of anyone in the policy, show warning to yourself that you’ll be replaced w/ policy owner
            1. Note: No warning if you’re `overLimitForwardsTo` or rule approvers (yet), right?
2. Deleting a workspace
    1. Only allowed by workspace Owner

I'll look up the warning message(s) which I think we agreed on in slack, but are the above statements correct? 😅

**Comment by trjExpensify** [comment]
Hm, why wouldn't we let the current manager of any report leave the workspace? Instead of showing the warning to the admin removing you in that case, we show the warning to the manager leaving.

**Comment by Beamanator** [comment]
Huh I might be crazy but i thought we already prevent people from leaving workspaces in that case or some related case, in OldDot - I'll see if I can find what I'm thinking of

**Comment by Beamanator** [comment]
Ok I was off a bit - this is what i meant to mention, which is actually "we don't allow removing the default approver from the workspace"

![Screenshot 2024-07-03 at 12 26 06 PM](https://github.com/user-attachments/assets/b503e882-55cd-4ec4-9ec3-ad68489caaee)

**Comment by trjExpensify** [comment]
Right, which you're already changing to be a confirmation before updating the default approver to be the workspace admin right? 

So to condense this down and bring a consistent implementation across leave/remove and show appropriate errors and confirmations on the NewDot side, I think we can summarise it as follows:

**Can leave and can be removed:**

- Any member
- Any approver 
- Any non-owner admin

**Can't leave or can't be removed:**
- Workspace owner
- Anyone with an outstanding `processing` report waiting on them to approve
- Anyone on a domain that has "prevent workspace creation/removal" enabled

**Confirmation modal suggestions:** 

- If the person _trying to leave_ is an approver, prompt with an alert modal warning on clicking `Leave`

> **Leave workspace** 
> If you leave this workspace you will be replaced in the approval workflow with Anthony Davies, the workspace owner. 
> 
> [Leave workspace] <--- red button
> [Cancel]

- If the person _being removed_ is an approver, prompt with an alert modal warning on clicking `Remove from workspace`.

> **Remove from workspace**
>If you remove Bob Williams from the workspace, we'll replace them in the approval workflow with Anthony Davies, the workspace owner. 
>
>[Remove] <--- red button
>[Cancel]

- If the person _trying to leave_ has an outstanding `processing` report to action on:

> **Leave workspace**
>Please approve any outstanding expense reports submitted to you before leaving the workspace.
> [Got it] <--- Green button


- If the person _being removed_ has an outstanding `processing` report to action on:

> **Remove from workspace**
>Bob Williams has outstanding expense reports to approve. Ask them to approve, or take control of the reports before removing them from the workspace.
> [Got it] <--- green button

____

Additionally, I think there might be a a few more cases to look into to add to the above to round it out completely and not run into errors on NewDot.

1. The designated reimburser on the workspace
2. The technical contact on the workspace 
3. The preferred exporter (when connected to an accounting solution)

Feel free to sense check those, but I think we'd handle them as follows:

For 1, let's prevent them from leaving/being removed as there isn't a clean "update to the workspace owner" when leave/removed, as the workspace owner could not have the reimbursement account shared with them. 

**Confirmation prompts**

- If the person _trying to leave_ is the reimburser, prompt with an alert modal warning on clicking `Leave`

> **Leave workspace** 
>You can't leave this workspace as the reimburser. Set a new reimburser in Workspaces > [your workspace] > Workflows, and try again.
> 
> [Got it] <--- green button

- If the person _being removed_ is the reimburser, prompt with an alert modal warning on clicking `Remove from workspace`.

> **Remove from workspace**
>You can't remove Bob Williams from the workspace. Set a new reimburser in Workspaces > [your workspace] > Workflows, and try again.
>
>[Got it] <--- green button

For 2 & 3, seems straightforward to update to the workspace owner upon leave/removed, because the owner will always be eligible as an admin to take on this role as the technical contact or preferred exporter.

**Confirmation prompts**

- If the person _trying to leave_ is the technical contact or preferred exporter, prompt with an alert modal warning on clicking `Leave`

> **Leave workspace** 
> If you leave this workspace you will be replaced as the [technical contact || preferred exporter] with Anthony Davies, the workspace owner. 
> 
> [Leave workspace] <--- red button
> [Cancel]

- If the person _being removed_ is the technical contact or preferred exporter, prompt with an alert modal warning on clicking `Remove from workspace`.

> **Remove from workspace**
>If you remove Bob Williams from the workspace, we'll replace them as the [technical contact || preferred exporter] with Anthony Davies, the workspace owner. 
>
>[Remove] <--- red button
>[Cancel]

**Comment by Beamanator** [comment]
> Right, which you're already changing to be a confirmation before updating the default approver to be the workspace admin right?

I believe you're talking about https://github.com/Expensify/Expensify/issues/404210 here, right? So this should already be done

**Comment by Beamanator** [comment]
So far I love your `Confirmation modal suggestions:` section 👍 

I think it mayyyy be missing a unique message for the case "Anyone on a domain that has "prevent workspace creation/removal" enabled"?

**Comment by Beamanator** [comment]
And as for the designated reimburser / technical contact / preferred exporter (when connected to an accounting solution) I also like those suggestions 👍

**Comment by Beamanator** [comment]
So how do we want to split this up? 😅  I feel like we need to:
1. Audit OldDot to check what we currently show in these scenarios
    - If different from what we newly expect (from @trjExpensify 's suggestions above), discuss what's best
2. Similar Audit for NewDot (some cases may not be possible in NewDot yet like domain stuff)
3. Clearly write up what all needs to be done in this issue, OR split into multiple implementation issues
    - Most likely the NewDot error messages should be workable externally / by C+

**Comment by garrettmknight** [comment]
## Non-owner admins:
OldDot steps to replicate:

1. Create policy in NewDot
2. Invite another user as admin
3. Log in as that admin
4. Open workspace members page & try to leave the workspace

**OldDot flow currently:** No option to leave workspace
**NewDot flow expected:** Non-owner admin can leave the workspace

## The designated reimburser on the workspace
**OldDot steps to replicate:**

1. Create policy in NewDot
2. Invite User B as admin
3. Connect a VBA
4. Go to Workspace > Reimbursement, select 'Direct'
5. Set User B as the Reimburser
6. Try to remove User B from the workspace in the Members page (to mirror them leaving the workspace themselves)

**OldDot flow currently:** User receives this message:
![image](https://github.com/user-attachments/assets/3c534cd2-8700-4a4d-a68f-f62540a00348)

**NewDot flow expected:** User should receive this updated message in an alert modal with action buttons:

>**Cannot Remove The Payer From This Workspace**
>To remove User B from this workspace, please go to **Workflows** and change the **Payer** in the **Make or track payments** settings to another admin.

## The technical contact on the workspace
**OldDot steps to replicate:**

1. Create policy in NewDot
2. Invite User B as admin
3. Go to Workspace > Connections
4. Set User B as the Technical Contact
5. Try to remove User B from the workspace  (to mirror them leaving the workspace themselves)

**OldDot flow currently:** User receives this message:
![image](https://github.com/user-attachments/assets/5957beb1-526b-4eea-8f0d-705d2dfded29)

**NewDot flow expected:** User should receive the same message in an alert modal with action buttons:
![image](https://github.com/user-attachments/assets/bf9dacc4-4630-4f18-9533-62f1deadfa27)

**Note that we haven't added 'Technical contact' in NewDot yet so this is reverting to OldDot**

## The preferred exporter (when connected to an accounting solution)
**OldDot steps to replicate:**
Create policy in NewDot
Invite User B as admin
Go to Workspace > Connections
Connect to QBO and select User B as the Preferred Exporter in the Export tab of the connection.
Try to remove User B from the workspace (to mirror them leaving the workspace themselves)

**OldDot flow currently:** User receives this message:
<img width="834" alt="Screenshot 2024-08-09 at 11 43 26 AM" src="https://github.com/user-attachments/assets/edbcfc97-d396-403c-b166-8069e943b941">

**NewDot flow expected:** User should receive this updated message in an alert modal with action buttons:

>**Preferred Exporter Updated**
>User B has been removed and replaced by the workspace owner, gknight@expensifail.com, as the Preferred Exporter for this workspace. To choose a different Preferred Exporter, go to the Accounting tab and click on the Export button for your accounting connection.

**Comment by johncschuster** [comment]
Back off, Melv! This is still being discussed!

**Comment by garrettmknight** [comment]
Ok, I made a few small tweaks and now I think @trjExpensify's [comment](https://github.com/Expensify/App/issues/43508#issuecomment-2258574485) above reflects what we're looking for here and is clear enough for an engineer to pick up. @Beamanator assigning you to start with the BE, but you'll wanna smack the `External` label on here when you're ready for the FE work.

**Comment by Beamanator** [comment]
Planning to take this on a bit later this week, as we have some more critical #wave-collect-finishing issues to focus on this week

**Comment by garrettmknight** [comment]
Taking Alex off this one since we have some higher priority bugs to handle in this project ahead of this polish.

**Comment by johncschuster** [comment]
Thanks for the update, @garrettmknight! I'll keep this bumping until it's ready to be picked up.

**Comment by johncschuster** [comment]
Bumping for Melvin. We still intend to work on this one.

**Comment by johncschuster** [comment]
Bumping for Melv

**Comment by johncschuster** [comment]
Bumping for Melvin

**Comment by trjExpensify** [comment]
@Beamanator @garrettmknight where is this one on the remaining list of items for the simple AA in NewDot project out of curiosity?

**Comment by Beamanator** [comment]
It's on the list, but a bit lower priority AFAIK - I am very happy to jump into this once I'm done with my current issues

**Comment by johncschuster** [comment]
Bumping for Melvin while we wait for this to get picked up 🙏

**Comment by johncschuster** [comment]
Bumping for Melv. This hasn't jumped in priority yet

**Comment by johncschuster** [comment]
Still lower priority

**Comment by johncschuster** [comment]
@garrettmknight can I get a sense check from you? Has the priority changed for this one?

**Comment by garrettmknight** [comment]
I still think this is low priority, but worth keeping open. If you want you can drop off the issue.

**Comment by johncschuster** [comment]
Thanks! I'm happy to stay on as the BZ 👍  Just wanted to get a sense check so I could apply the right level of urgency. I'll keep it bumping!

**Comment by johncschuster** [comment]
Happy Monday! I'm back with another Melvinbump.

**Comment by johncschuster** [comment]
Bumping for Melv 👍

**Comment by johncschuster** [comment]
Bumping again to keep Melv at bay

**Comment by johncschuster** [comment]
I'm OOO starting Monday, December 23, and will be returning Monday, January 6. I'll keep this bumping then!

**Comment by johncschuster** [comment]
Hello everyone! I'm back from break! 👋

**Comment by johncschuster** [comment]
Okay, it looks like I'm the only assignee at the moment, and this issue is still low priority. I'm going to downgrade the label to `Monthly` for now and keep it bumping so it doesn't get eroded. 

We'll increase the priority again when the time is right 👍

**Comment by quinthar** [comment]
Can we expand this to allow *anyone* to leave (except the owner), including both admins *and* members?  More info here:

https://expensify.slack.com/archives/C06ML6X0W9L/p1737402644808479?thread_ts=1737153983.772019&cid=C06ML6X0W9L

**Comment by johncschuster** [comment]
Sounds good! I've updated the expected results in the OP as well as the repro/testing steps:

> ## Action Performed:
> 1. As `User A`, create workspace in NewDot 
> 2. Invite two other users. Promote `User B` to workspace admin, and leave `User C` as a regular user
> 3. Log in as `User B`
> 4. Open workspace members page & try to leave the workspace
> 5. Log in as `User C`
> 6. Open workspace members page & try to leave the workspace
> Expected Result: Both `User B` and `User C` should be able to leave the workspace
> 
> ## Expected Result:
> There should be a way for a workspace non-owner to leave a workspace.
> 
> ## Actual Result:
> There's no way for either of the workspace non-owners to leave the workspace.

**Comment by johncschuster** [comment]
Ah! Jason made a great callout, [here](https://expensify.slack.com/archives/C06ML6X0W9L/p1737403715018199?thread_ts=1737153983.772019&cid=C06ML6X0W9L). I will need to update the OP to account for the following scenarios:

- Admins who are locked to policies via domain control
- Admins who are the designated reimbursers
- Admins who are the designated technical contact

**Comment by trjExpensify** [comment]
Yeah, and let's make sure [we handle them appropriately](https://github.com/Expensify/App/issues/43508#issuecomment-2258574485).

**Comment by dangrous** [comment]
Hi! Wanted to sync up on this issue vs. [this issue](https://github.com/Expensify/App/issues/55462#issuecomment-2601357364) since it seems like they're basically the same. Should we consolidate?

**Comment by trjExpensify** [comment]
Yeah, I think consolidate makes sense. I wrote out some thoughts on the different cases and handling them a while back in [this comment](https://github.com/Expensify/App/issues/43508#issuecomment-2258574485).

**Comment by johncschuster** [comment]
It looks like we may not be consolidating these issues after all (per [this comment](https://expensify.slack.com/archives/C06ML6X0W9L/p1738712798523389?thread_ts=1738711479.457309&cid=C06ML6X0W9L) in Slack)

**Comment by dangrous** [comment]
@MonilBhavsar  do you have a moment to look at this? It looks like the functionality already exists (I saw you made the LeavePolicy command in Auth), just wasn't implemented fully in New Dot? Trying to sort out these long chains of comments. Thanks!

**Comment by MonilBhavsar** [comment]
> I think it's just showing/not showing the "leave" button in the appropriate situations, right? No backend work?

That sounds right.

We had an option to Leave policy in the three dot menu
Please see video in test section in this PR https://github.com/Expensify/Auth/pull/10245

That seems to be removed somehow, so yes we need to add a button and call the API. That should hopefully do it

**Comment by trjExpensify** [comment]
Ah, FYI, I just commented on the [app PR](https://github.com/Expensify/App/pull/56316#issuecomment-2653092050).

**Comment by trjExpensify** [comment]
I'm going to take over managing this, and consolidating https://github.com/Expensify/App/issues/43508 and https://github.com/Expensify/App/issues/55462 to https://github.com/Expensify/App/issues/56750.
